hazelcast-tpc-engine/src/test/java/com/hazelcast/internal/tpc/EventloopTypeTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.tpc;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class EventloopTypeTest {
+
+    @Test
+    public void test_fromString(){
+        assertEquals(EventloopType.NIO, EventloopType.fromString("nio"));
```

Comment:
nit: you may add assertions for upper-cased strings as well, as those are supported

Reply:
Done